### DISCUSSION TITLE
unminimize from minimized state

### DISCUSSION THREAD

**Discussion by Cruising0904** [WayfireWM/wayfire#1721] [Q&A]
After trigger minimized action, no way to restore from it except using fast-switcher. And I can't know if it's on focused, it seems that focus is on other window. It seems like the feature is not implemented I think, however is there a way to restore it? configuring window-rule or etc.
![image](https://user-images.githubusercontent.com/29267295/219367492-84914110-ad00-4f38-bf6b-d1472c6461bd.png)
![image](https://user-images.githubusercontent.com/29267295/219367558-3b449e1f-b41a-462c-9b13-c61513110576.png)
![image](https://user-images.githubusercontent.com/29267295/219367633-175a87ef-d4b9-4c2d-bd15-a555f3d4cd2c.png)

2. Is there a way to have a screen pre-sized window in spot with a shortcut? I was trying to playing with window-rule but never made it. What I would want is like press minimize once make a window to be a certain size in certain spot, on next press minimize it, on third restore it to first state.

cheers to wayfire :)

**Comment by Cruising0904** [comment]
oh, and I wonder what is sticky state does. is it differentiate with always on top?

**Reply by soreau** [reply]
> oh, and I wonder what is sticky state does. is it differentiate with always on top?

Sticky means show the window on all workspaces.

**Reply by Cruising0904** [reply]
now I see..  thanks for the answer : -)

**Comment by soreau** [comment]
> After trigger minimized action, no way to restore from it except using fast-switcher. And I can't know if it's on focused, it seems that focus is on other window. It seems like the feature is not implemented I think, however is there a way to restore it? configuring window-rule or etc.

You can restore windows using wf-panel from wf-shell, with the window-list widget enabled.

> 
>     2. Is there a way to have a screen pre-sized window in spot with a shortcut? I was trying to playing with window-rule but never made it. What I would want is like press minimize once make a window to be a certain size in certain spot, on next press minimize it, on third restore it to first state.

That's not what minimize is for. You probably want to look into the grid plugin. It allows you to use a keybinding to resize windows to a corner, edge or maximize state.

> 
> 
> cheers to wayfire :)

**Reply by Cruising0904** [reply]
umm... I omitted some explanation.. actually I set geometry on created.. also set grid config. I am not sure about wf-panel is what I am thinking of though, I want extra slot(there might be but not that I know of) in center like floating by pressing a defined key combination.
I want maximize and float window also minimize... sorry if it's not understandable if I can I will upload my config to show you how I set

**Reply by Cruising0904** [reply]
When I used wf-shell on start up it was significantly laggy though ... is it now usable?

**Comment by Cruising0904** [comment]
If I change slot_c resolution maybe possible for my desire.. the problem is I don't know how to do this
